mkdocs.yml

```diff
@@ -0,0 +1,87 @@
+# Project Information
```

Comment:
This file should be removed since we already have one under doc/.
This patch should also be pushed to master please.